cinch/roles/jenkins_master/tasks/post_configure.yml

```diff
@@ -20,9 +15,24 @@
   # These scripts are run in the order listed below, so please edit this list
   # using the desired execution order.
   with_items:
-    - "{{ lookup('template', role_path + '/templates/set_executors.groovy') }}"
-    - "{{ lookup('template', role_path + '/templates/set_usebrowser.groovy') }}"
-    - "{{ lookup('template', role_path + '/templates/set_slaveport.groovy') }}"
+    - script: "{{ lookup('template', role_path + '/templates/set_executors.groovy') }}"
+    - script: "{{ lookup('template', role_path + '/templates/set_usebrowser.groovy') }}"
+    - script: "{{ lookup('template', role_path + '/templates/set_slaveport.groovy') }}"
+    - script: "{{ lookup('template', role_path + '/templates/user.groovy') }}"
+      when: "{{ jenkins_security_enabled }}"
+    - script: "{{ lookup('template', role_path + '/templates/roles_and_ldap_auth.groovy') }}"
+      when: "{{ jenkins_security_enabled }}"
+  become: true
+  become_user: "{{ jenkins_user }}"
+  when: (item.when is not defined or item.when) | bool
```

Comment:
Oh, you mean that boolean logic that took me at least 7 tries to get right? Yeah, let me toss a comment on that one.